core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerRule.java

```diff
@@ -202,7 +202,7 @@ static CircuitBreakerRule of(CircuitBreakerRule... circuitBreakerRules) {
     /**
      * Returns a {@link CircuitBreakerRule} that combines the specified {@link CircuitBreakerRule}s.
      */
-    @SuppressWarnings("OptionalGetWithoutIsPresent")
+    @SuppressWarnings({ "OptionalGetWithoutIsPresent", "NullAway" })
```

Comment:
After upgrading Guava, it seems that NullAway incorrectly infers nullness for `Iterables.get()`